test/code/plugins/oms_common_test.rb

```diff
@@ -94,6 +100,256 @@ def teardown
       "OSAlias=UniversalR\n" \
       "OSManufacturer=Central Logistics GmbH\n"
 
+    # Begin:  Unit Tests for some simple methods.
```

Comment:
This entire test file is all unit tests; this comments don't clarify much.

Reply:
Trying "Public Internals, which is closer to what I labeled the code.  

My tendency is to collate things in alphabetical order, or functional order if that ends up being more helpful.  I may not be familiar enough with the domain, but I didn't see an existing order, and wanted to categorized somehow.  Were I refactoring these tests, I would literally put every one in alphabetical order, and probably supplement with some glossary on top of area categories, but if I could just understand an existing categorization I'd be more comfortable.